src/test/java/com/gooddata/md/report/GridElementTest.java

```diff
@@ -1,36 +1,46 @@
 /*
- * Copyright (C) 2007-2014, GoodData(R) Corporation. All rights reserved.
+ * Copyright (C) 2007-2016, GoodData(R) Corporation. All rights reserved.
  */
+
 package com.gooddata.md.report;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import org.testng.annotations.Test;
 
-import java.io.InputStream;
+import java.util.ArrayList;
 
 import static com.gooddata.JsonMatchers.serializesToJson;
+import static com.gooddata.md.report.MetricGroup.METRIC_GROUP;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.CoreMatchers.notNullValue;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+import static org.hamcrest.Matchers.instanceOf;
 
 public class GridElementTest {
 
-    public static final String URI = "/ELEM_URI";
-    public static final String ALIAS = "ELEM_ALIAS";
-
     @Test
-    public void testDeserialization() throws Exception {
-        final InputStream is = getClass().getResourceAsStream("/md/report/gridElement.json");
-        final GridElement element = new ObjectMapper().readValue(is, GridElement.class);
-        assertThat(element, is(notNullValue()));
-        assertThat(element.getUri(), is(URI));
-        assertThat(element.getAlias(), is(ALIAS));
+    public void testSerializer() throws Exception {
```

Comment:
as serializers are not contained in the interface any more, please extract the test as well